pkgs/development/coq-modules/compcert/default.nix

```diff
@@ -0,0 +1,125 @@
+{ lib, fetchzip, mkCoqDerivation, coq, flocq, compcert
+, ocamlPackages, fetchpatch, makeWrapper, coq2html
+, stdenv, tools ? stdenv.cc
+, version ? null
+}:
+
+with lib;
+
+let compcert = mkCoqDerivation rec {
+
+  pname = "compcert";
+  owner = "AbsInt";
+
+  inherit version;
+  releaseRev = v: "v${v}";
+
+  defaultVersion =  with versions; switch coq.version [
+      { case = range "8.8" "8.11";  out = "3.7"; }
+      { case = range "8.12" "8.13"; out = "3.8"; }
+    ] null;
+
+  release = {
+    "3.7".sha256 = "1h4zhk9rrqki193nxs9vjvya7nl9yxjcf07hfqb6g77riy1vd2jr";
+    "3.8".sha256 = "1gzlyxvw64ca12qql3wnq3bidcx9ygsklv9grjma3ib4hvg7vnr7";
```

Comment:
Where is `3.9` gone?

Reply:
Oops.

Reply:
Fixed and now the default unqualified attribute `compcert` is set at version 3.9 as before.